presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/WriteMapping.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class WriteMapping
+{
+    public static WriteMapping booleanMapping(String dataType, BooleanWriteFunction writeFunction)
```

Comment:
I'd rename these like `booleanWriteMapping` (consistent with `booleanReadMapping`) and static import them everywhere.

Reply:
See my answer to https://github.com/prestosql/presto/pull/109#discussion_r254917898